src/main/java/com/airdnb/clone/domain/stay/entity/Amenity.java

```diff
@@ -0,0 +1,22 @@
+package com.airdnb.clone.domain.stay.entity;
+
+import lombok.AccessLevel;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+
+@Getter
+@NoArgsConstructor(access = AccessLevel.PROTECTED)
+public class Amenity {
+
+    private String mainCategory;
+    private String name;
+    private String description;
+
+    @Builder
+    public Amenity(String mainCategory, String name, String description) {
+        this.mainCategory = mainCategory;
+        this.name = name;
+        this.description = description;
+    }
```

Comment:
모든 필드를 대상으로 `@Builder` 를 사용하기 때문에 클래스 레벨에 붙이는걸 추천합니다

```suggestion
    @Getter
    @NoArgsConstructor(access = AccessLevel.PROTECTED)
    @AllArgsConstructor(access = AccessLevel.PROTECTED)
    @Builder
    public class Amenity {
```